lib/board.rb

```diff
@@ -0,0 +1,64 @@
+require 'pry'
+class Board
+attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
+
+  def reset!
```

Comment:
This is the method I would create the array with empty spaces. 
`@cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]`